packages/react/babel.ts

```diff
@@ -51,10 +52,20 @@ module.exports = function (api: any, options: NxReactBabelOptions) {
   };
 };
 
-function getReactPresetOptions({ presetOptions, env }) {
+function getReactPresetOptions({
+  presetOptions,
+  env,
+}: {
+  env: string;
+  presetOptions: NxReactBabelOptions;
+}) {
   const reactPresetOptions: Record<string, string | boolean> = {
     runtime: presetOptions.runtime ?? 'automatic',
-    development: env !== 'production',
+    development:
```

Comment:
I'd rather keep the option `development` to match the `@babel/preset-react` option, so it's clear that it is an override.

So something like:

```js
development: presetOptions.development ?? env !== 'production'
```